server/middleware/validation.js

```diff
@@ -280,6 +282,14 @@ const validators = {
             }
         }
         return message || validationErrors.oneOf();
+    },
+    isValidWithinGivenDays({ label, value, message, props }) {
+        let limitDate = new Date();
+        limitDate.setDate(limitDate.getDate() + parseInt(props));
```

Comment:
Would props _only_ contain a value for this, or could there be potential it would have more fields? In which case you'd access the property on the object instead

Reply:
The props can be anything, it'll be passed in on the validator object so if people need this sort of thing in the future they can use props in whatever way works for them